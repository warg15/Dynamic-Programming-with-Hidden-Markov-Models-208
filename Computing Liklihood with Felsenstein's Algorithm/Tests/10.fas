>American-flamingo
GCAACTATATACTGAAGACGATAAATATATCCCTATGGAGACAACAGCAGGTGCTATGCAGGCTAATCGTCTGATCCATGATCGTAAGTGCAACATTTCGATATCAATCTTACAATATTATACAATCGGTGATACATTTATGGATATAATGTTAGTTCGATACATATGATTACCTTTAGCGAATTCTATTCAAGAAACTG
>Budgerigar
GCGACTATATACTGAAGACGATGAATATATCCCCATGGAGACAACAGGAAGTGCTATGTAGGCTTATCACTTGATCCATGACCGTAAGCGCAACACTTAGATTTCAATCTTACAATGCTATACAATTGGTGGTACATTTATAGATATAATGTTAGTTTAATACATATAGTTACCCTTAGTGAATTCTATTCAAGAAACTA
>Northern-fulmar
GTAACGTTATACCGAAGACGATAAATATATCCCTATGGAGACAATAGCAGGTGCTACACAGGCTAATCACTTGGTCCACGATCGTAAGTGCAACATTTCGATATCAATCTTTCAACGTTATACAATCGGTGATACGTTTATGGATATAATGTTAGTTTGATACATATAATTACCTTTAGCGAATTCTATTCAAGAAACTA
>Red-legged-seriema
GTACCTATATACTGAAGACGATAAATATATCCCTATGGTGACAACAGCAAGTGCTATGCAGGCTAATCACTTGATCCGTGATCCTAAGTGCAACATTTCGATATCGATATTTCAATGTTATACAATCGGTGATACATTTATGCATATAATGTTAGTTTGATACATATAATTACCTTTAGCGAATTCTATTCAAGAAACTA
>Adelie-penguin
GTAACTATATACTGAAGACGATAAATATATCCCTATGGAGACAACAGCAGGTGCTACGCAGGCTAATCACTTGATCCACGATCGTAAGTACAACATTTCGATATCAATCTTCAAACGTTGTACAATCGGTGATACATTTATGGATATAAAGTTAGTTTGATACATGTAATTACCTTTAGCGAATTCTATTCAAGAAACTA
>Great-crested-grebe
GTAACTATATACTGAAGACTATAGATATATCCCTATGGAGACAACAGCAGGTGCCATACAGGCTAATCACTCTATCCATGATCGTAAGTGGAACATTTCGATATCAATCTTACAATATTATACAATCGGTGATCATTTTATGGATATAATGTTAGTTTGATACACAAAATTACCTTTAGCGAATTCTATTCAAGAAACTG
>Peregrine-falcon
GTAACTATATACTGAAGACGATAAATATATCTCTATGGAGACAACAGCAGGTGCTATGCAGACTAATCACTTGATCCATGGTCGTAAGTGCCACATTTCAATATCAATCTCTCGATGTTATACAATCGGTGAGACATTAATGGATATAATGTTAGTTTGGTACACATAATTACTTTTAGCGAATACTATTCAAGAAACTA
>Dalmatian-pelican
GTAACTATATACTGAAGACGATAAATATATCCCTATGGAGACAACAGAAGGTGCTACGCAGGCTAATCACTTGATCCACGATCGTAAGTGCAACTTTTCGATATCAATCTTTCAACGTTATACAATCGGTGATACGTTTATGGCTATAAGGTTTGTTTGATACATATCATTGCCTTTAGCGAATTCTATTCCAGAAACTA
>Speckled-mousebird
GTAACTACGTACTAAAGACAATAAATATATCCCTATGGAGACAACAGAAGGTGCTATGCAGGCTAATCACTTAATCTATGATCATAATTGCAACATTTTGATATCAATCCTGCAATGTTATACAATCGGTGATACATCTGTGGATATAATGTTAGTTTGATACGCAAAATTACCTTTAGCGAATTCTATTCAAGAAACTA
>Little-egret
GTAACTATATACTGAAGACGATAAAGATATCCCTCTGGAGACAAAAACGGGTGCTACGCAGGTTAATCACTTGATCCACGATCGTAAGTGCAACATTTCAATATCAATCTCTCAATGTTATACAATCGGTGATACACTTACGGATATAACGTTAGTTTGATACATATAATTACCTTTAGCGAACTCTTTTCCAGAAACTA
>Rifleman
ATAACAATATACTGAGAACAATGAATATATGCCTGTGGAGACAACAGCAGGTGTTATGTAGGCTAAGTACTTGATCCGTGACCGTAAGTGCAACGTTTCGAGATCAATCTTTCAATGTTATACAATCGATGATGCATCTATGGATATCATGTTAGTTTGATTCATATAATTACCTTTAGCGAATCCTGTTCAAGAGACTA
>White-tailed-eagle
GTAACTATATACTGAAGACGATAAATATATCCCTATGGAGACAATAGTAGGTGCTATACAGGCTAATCACTCGATCAATGATTGTAAGTGCAACATTTCGATTTCAATATTTCAGTGTGGTACAATCGGTGATACATTTATGGTTATAATGTTAGTTTGATACATATAAATACCTTTAGCGAATTCTATTCAAGAAACTA
>Downy-woodpecker
GTAACTATATACCGGGGACGATAAATATATTTTTGTTGAGACTACAACAGATGCCATCCAGGTTAATCACCTGATTCATGATCGCAGGTGCATCATTTTGATACCAATCTGGCGATGTAACGCAATCGGTGATACATTTATGGATATAATGTTAGTTTGATACGTATAATTAGTTTTAACAAACTCTATTCAAGAAACTA
>Hoatzin
GTAGCTATATACTGAAGACGATAAGAATATTCCTATGGAGACAAGAGCAGGTGCTATGCAGTCCAATCATTTGATCCATGAACGTAAGTGCAGCATTTCGATATCAATCTTTCACTGTTATACAATCGGTGATACTTTTATGGATATAATGTTAGTTTGATACATATAATCGCCTTTAGCGAATTCAACTCAAGAAACTA
>Chuck-wills-widow
GTAACTATATACTAAAGACGATAAATATATCCCTATGGAGACAACTACAGGTGCTATGCAGGCTGATCACTTGATCCATAATCGTAAGTGCAACATGTCGATATCAATCCTTCAATATTATACAATCGGTGATACATTTATGGATATAATGTTAGTTTGATACATATAATTACCTATAGTGAACTCTATTCAAGAAAATA
>Kea
GCGACTATATACTGAAGACGATAACTATATCCCCATGGAGACTACCGGAGGTGCTATGCAGACTAATCACTTTATCCATGACCGTAAGTGCAACATTTCGATTTTAATCTTACAATTTTAGACAATTGTTGGTACATTTATGGATGTAATGTTAGTTTAATACATATAATTACCTTTAGCGATTTCTATTCAAGAAACTA
>Golden-collared-manakin
GTAATCGTGTACCGAAAACAATGAATATATCCCTGTGGAGACACCAGCAGGTGCTATGTAAGCCAATCGTTCGATCCGTGACCGTACTTATAACGTTCCGATTTCCATCTTTCAATGTTATACAATCGGTAATACATCTATGGATATAATGTTAGTTTGATTCATATAATTACCTTTCGCGAATCCTGTTCAAGAATCTG
>MacQueens-bustard
TTAACTATATACTGAAGACGATAAATATATGCCTCTGGAGACAATAGCAGGTGCTGTGCAGGCTAATCACTTGATCCGTGATTGTAAGTGCAACATTTCGGTATCAATCCTTCAATATTATACGATCGATGATACATTTATGTATATAATGTTAGTTTGATACATATAATTACCTATAATGACCTCTCTTCAACAAACTA
>Bar-tailed-trogon
TAAACTGTATACTGAAGACGGTAAATACATCCCTATGGAGACAATAGAAGGGGCTATGCGGGCTAATCACTTGATCCATGATCGTAAGTGCAACGTTTCGATATCAATCTTTCAATGTTATACAATCGGTGACACATTTGTGGATGTAATGTTGGTTTGATACCTATAATTACCTTTACCAAGTTCTATTCAAGAAACTA
>Chicken
TTAACTATATATTGATGACGATAAAGATATCCCTATGGAAACAACAGCAGGCGCTATGCGGGCTAAACACTCGATCCATGATCGTAAAAGCAACATTTCGATATCAGTCTTTCAATGTAATACAAACGGTGATAAATGTATGACTATGATATTGGTTTGATACATATGCTTACCTCCAGCGAATTCTATTCAAGAGACTA
>Medium-ground-finch
GTAACCATATAGCGAAGAGGATGAATATATCCCTGCGGAGACAACAACAGGAGCTATCTAGGCCAATCGCTCGATCTGTGACCGTAATTGTAACGTTTCGATTTCAATCTTTCCATGTTATACAATCCGTGTTATATCTGAGGATATAATGTTAGTTTGATTCATATAATTACCTTTAGCGAATCCTGTTCAAACATCTA
>Emperor-penguin
GTAACTATATACTGAAGACGATAAATATATCCCTATGGAGACAACAGCAAGTGCTACGCAGGCTAATCACTTGATCCACGATCGTAAGTACAACATTTCGATATCAATCTTCAAACGTTATACAATCGGTGATACATTTAGGGATATAAAGTTAGTTTGATACATGTAATTATCTTTAGCGAATTCTATTCAAGAAACTA
>Bald-eagle
GTAACTATATACTGAAGACGATAAATATATCCCTATGGAGACAATAGTAGGTGCTATACAGGCTAATCACTCGATCAATGATTGTAAGTGCAACATTTCGATTTCAATATTTCAGTGTGGTACCATCGGTGATACATTTATGGTTATAATGTTAGTTTGATACATATAAATACCTTTAGCGAATTCTATTCAAGAAACTA
>Rhinoceros-hornbill
GTAACTGTATACTGAAGATGATAAATAGAGCCCTATGGGGAAAACAGCAGGTGCTGTGCAGGCTAATCACTTGCTCCAGGGTCGTCCGTGCAACATTTCGATATCAATCTTTCAATGTTATACAATGGGCGATACATTTATGGATATAATGTTAGTTTGAGACGTATAATTGCCTTTAGCGAATTATGTTCAAGAAACTA
>Carmine-bee-eater
TTAACTATATACTGAAAACGATAAATATATTGCTATGGAGACAACAACAGGTGCTATGCAGGCTAATCAGCTGACCCATAATCGTAAGTGCAGCACTTAGAGATCAATTTTTCAATGTTATACAAGCGGCGATACATTTATCGATTTAATGCTAGTTTGATACGTATAATTACCTTTAGCGAATTCTATTCAAGAAACTA
>Barn-owl
ATAACTATATACTAAAGACGATATATATATCCCTATGGAGACAACAGCAGGTGCTATGCAGTCTAATAACTTGATCCATGATCGTAAGTGCAACGTTTCAATATAAATCTTCCAATGTTATACAATCGGTGACACATTTATGGATATAATGTTAGCTTGATACATATAATTACCTTTAGCGAATTCTAGTCGAGAAACTA
>Annas-hummingbird
ATGACCATATACTGAAGACGATAAATATATCCCTGTGGGGACAACAGGAGGTGCTATGCGGGCTAATCATATTATCCATGATCATAAGCGCAACATTTCGAGATCAATCCCCCAATATTATGCAATCGGTAATACATTTATGGATATAATGTTAGTTTGATATATATAATTACCTATAGCGAACTCTATTCAAGAAACAA
>Common-ostrich
GTAACTATAGACTGACGACGATAAATATATCTCTATGGAAACAACAGCTGGTACTATGCAGGTTAATCACCTGATCCGTGATCGTCAGTCCAACATTTCGATATCAATCTTTCAATATTATAGAATCGATGATACATGCATGGAGAAGATTTTAGTTTAATACATACATTTAGCTCTATCGAACCTTATTCAAGAAACTA
>Zebra-finch
GTAACCATATAGCGAAGAGGATGAATATATCCCGGCGGAGACAACAACAGGTGCTATCTAGGCTAATCGCTCGATCTGTGACCGTAATTGTAACGTTTCGATTTCAATCTTTCCATGTTATACAATCTGTGTAACATCTATGGATATAATGTTCGTTTGATTCATATAATTACCTTTAGCGAATCCTGTTCAAGCATCTA
>Red-throated-loon
GTAACTATATATTGAAGACGATAAATATATCCCTATGGATACAACAGCAGGCGCTATGCAGGCTAATCACTTGATCCACGATCGTAAGTGCAACATCTCGATATCAATCTCTCAACGTGATACAATCGGGGATACATTTATGGATATAATGTTAGTTTGATACATATAATTACCTTTAGCGAATTCTATTGAAGAAACTA
>Crested-ibis
GTAACTATATACTGAAGACGATAAATATATCCCTATGGAGACAACAGCAGGTGCTACGCAGGCTAACTACTTGATCCACGATCGTAAGTGCAACATTTCGATATCAATCTTTCAACGTTATACAATCGGTGATATATTTATGGATATAATGTTAGTTTGATACATATAATTACCTTTAGCGAATTCTAGTCCAGAAACTA
>Grey-crowned-crane
GTAACTATATACTGAAGGCGATAAAGATATCCCTATGGAGACAACAGCAGGTGCTATGCAGGCTAATCACTTGATCCATGATCGTAAGTGCAACATTTCTATATCAATCTTTCAATTATATATAATCGGTGATACATTTATGGATATAATGTTAGTTTGATACATATAATTACCTTTAGTGAAGTCCATTCAAAAAACTA
>American-crow
GTAACCATATACCGAGGACGATGAATATATCCCTGTGGAGACAACAACAGGTACTGTGTAGGCTAATCGCTCGATCTGTGACCGTAATTGTAACGTTTCGATTTCAGTCTTTCCATGTTATACAGTCCGTGTTACATCTATGGATATAATATTAGTTTGATTCATATAATTACCTTTATCGAATCCTGTTCAAGAATCCA
>Pigeon
GTAACTATGTACTAAAGACGATAAATATTTGTCCATGGAGAAAACAGCAAATGCTATGCAGGATAACCACTTGATCCATGATCGTAATTGCAACGTTTCGATATCAGTCCTTCAATATTATACAATCGGTGATACATTTACCGATATAATGTTAGTTTGATACATACACTTACCTTTAGCAAATTCTATTCAAGAAACTA
>Chimney-swift
GTGACTGTATACTGAATACAACAAATATATCCCTATGGGTACAACAGCATGTGCTATACAGACTAATCACATGATCCATGATCATAAGCGCAAGATTTCGAGATCAAGCCTTCAATATCATACAATCGGTGATACGTTTAGGGATATAATGTTAGGTTGACACATATAATTACCGATAGCGAACTCTATTCAAGAAAGTA
>Turkey
TTAACTATACACTGATGACAATAAAGATATCCCTATGGAAACAACAGCAGGCGCTATGCGGGCTAAACACTCGATCGATGATCGTAAAAGCAACATTCTGATATCAGTCATTCAATATTATACAAATGGTGATACATGTATTACTATGATATTAGTTTGATACATATGATTACCTCCCGAGAATTCCATTCAAGAGACTA
>Great-cormorant
GTAACTATATACTGAAGACGATAAATATATCCCTATGGAGACAACAGCAGACGCTACGCAGGCTAATCGCTTGATCCACGATCGTAAGTGCAACATTTCGATATCAATCTTTCAATGTTATACAATCGGTGATACATTTATGGATATAATGTTAGGTTGATACATATAGTTACCTTTAGCGAATTATATTCCAGAGACTA
>Cuckoo-roller
GTAACTATATACCGAAGACGATAAATATATCCCTATGGAGACAACAGCAGGTGCCATGCAGGCTAATCACTCAATCCATGATCGTAAGTGCAACATTTCGATATCAATCTTTCAATGTTATACAATCGGTGATACATTCATGGATTTAATAATAGTTTGATACGTATAATTACCTTTAGCGAATACTATTCAAGCAACAG
>Sunbittern
GTAACTATATACTGAAGACGATAGATATATCCCTATGGAAACAACAGAACGTGCTATGCAGGCTAATCACTTCATGCACGATCGTAAGTACAACATTTTGATATCAACCTTTCAACGTTATACAATCAGTGATACATTTATGGATATAATGTTAGTTTGATACATATAATTACCTTCAGCGAATTCTATTCAAGAAACAA
>Turkey-vulture
GTAACTATATACTGAAGACGACAAATATATCCCTATGGAGACAACAGCAGGTGTTGTGCAGGCTAATCACTTGATCCATGATCGTAAGTGCAACATTTCGATATTAATCTTTCAGTGTTATACAATCGGTGATACATTTATGGATATAATGTTAGTTTGATACATATAATTACCTTTAGCGAATTCTGTTCAAAAAACTA
>White-throated-tinamou
GTGGCTATACACTGATGACGATTAATGTATTTCTATGGAAATGACAGCCAGTACTATGCAGGGTATTGACCTGATCCATGACCGTAAGGGCACCACTTCGATAACACTCTTTCAATATTGTACACTCGATGATACATGCATGGATAGAGTTTTAGTTTAACACATGTATTTACCGCTAGCCGCTAGTATTCAAGAAATTA
>Brown-mesite
GTAACTATATACTGATGACAATAATTATATCCCTAAGGAGACAACAGCAGATGCGATACAGGCTAATCACTTGATCCATGATTGTAAGTGCAACATTACGATATCAATCTTTCGATATTAGACAATCGGTGGTACATTTATGGATATAATGTTGGTTTGATACATATACCTACTTTTAGCGAATTCTATTTAAGAAACCA
>Red-crested-turaco
GTAACTATATACTGAAGACGATAAATATGTCCCTATGGAGACAACAGCAGGTGATATGCAGGATAATCACTTAATTCATGATCGTGACTGCAACATTTCGATATCAATCCTTCAATATTATACAATCTGTGATACATTTATGGATATAATGTTAGTTTGATATATATAATTACCTTTAGCCAATTCTATTCAAGAAACTA
>White-tailed-tropicbird
GTAACTATATACTGGAGAACATAAATATATCTCTATGGAAATAACAGCAGGTGCTATGCAGGCTAATCACTTGATCCACGATCGTAAATGCAACATTTCGATATCAATCCTTCAACGTTATACGATCGCTGATACCTTTATGGATATAATGTTAGTTTGATACATATAATTACCTTTAGCGAATTCTATTCAAGAAACTA
>Common-cuckoo
GAAACTACGTACTGAGTACGGTAAATGTAGCCCTATGGGGACAACAGCAGGTGCTATGCAGGCTAATCACTTGATCCATGATTATAAGTGCAACATGTCAGTATCCATCCTTCAATATCATACAATCGGTTCTACGTTTATGAATATAATGTTATTTTGGAATATATAATTACCTGTAGCGAATTCTACTCAAGAAATTA
>Pekin-duck
TGAACTATATACTGATCACAATAAATATATCCCTATGGAAACAACAGCAGACGTTATGCAGGCTGAACACTTGATCCATGATCGTAAGTGCAACATTTCGATATCAATTTTTCAATATTATACAATCGGTGATAGATATCTGGATATAATGTTTGTTTGATACATATAATTACCTCTAGCGAATTCTATTCAAAAAACTA
>Killdeer
GTAACTCAATACTGAAGACGATAAATATATCCCTATGGAAACAACAGCAGGTGCTATGCAGGCTGATCACTTGATCCGTGATCGCTAATGCAACATTTCAATAGCAATCTTTCAATGTGATACAATCGGTGATACATTTATGGATATAATGTTAGTTTGATACATATTATTATCTTTAGCGAATTCTATTCAAAAAGCTA
>Yellow-throated-sandgrouse
GTGACTATATACTGTTGACGATAAGTATATGCCTATGGAGACGACAGCAGGCGCTATGCAGGCTAATCACTTAATCCATGATCGTAAGTGCAACATTTCGATATCAATCTGTCAATGTTACACAATTGGTGATACGTTTAAGGACATAATGTTGGTTTGATACATATACCTACCTTTAGCGAATTCTATTCAAAAAACTA
