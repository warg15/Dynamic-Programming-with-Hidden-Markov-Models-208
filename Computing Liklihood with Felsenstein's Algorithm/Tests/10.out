-3487.668085953986
